src/components/Game/Game.js

```diff
@@ -139,26 +229,40 @@ class GameView extends Component {
             <main className={classes.Game}>
                 <Timer timeIsOver={this.timeIsOver} />
                 <Slots
-                    lettersInSlots={lettersInSlots}
+                    currentAnswer={currentAnswer}
                 />
                 <Backspace
                     clicked={this.handleBackspaceClick}
-                    disabled={lettersInSlots.length === 0 || isSubmitVisible}
+                    disabled={currentAnswer.length === 0 || isSubmitVisible}
                 />
                 {
                     !isSubmitVisible
                         ? (
                             <Letters
                                 letters={letters}
                                 clicked={this.handleLetterClick}
+                                dragDisabled={dragDisabled}
+                                dragEnd={this.handleDragEnd}
                             />
                         )
                         : (
                             <Submit
                                 onSubmit={this.onSubmit}
                                 onCancel={this.onCancel}
+                                showCancel={dragDisabled}
+                            />
+                        )
+                }
+                {
+                    !dragDisabled
+                        ? (
+                            <Submit
+                                onSubmit={this.onSubmit}
+                                onCancel={this.onCancel}
+                                showCancel={dragDisabled}
                             />
```

Comment:
exactly the same props are passed to `Submit` here and few lines above
it's not a blocker (can be addressed later) but perhaps it would be cleaner to modify conditions and use `Submit` component once